src/main/webapp/app/guided-tour/guided-tour.component.ts

```diff
@@ -25,13 +25,12 @@ export class GuidedTourComponent implements AfterViewInit, OnDestroy {
     // Sets the highlight padding around the selected .
     public highlightPadding = 4;
 
-    public currentTourStep: TourStep | null;
+    public currentTourStep: any;
```

Comment:
I had to change the type to `any` because I would receive the build errors `Property 'x' does not exist on type 'y'` when accessing properties of certain subclasses in the html template. The type checks whether the tour steps are correct are already done in the `GuidedTour` interface which is why I think that another check here is not mandatory.